src/common/components/organisms/FidgetTray.tsx

```diff
@@ -0,0 +1,42 @@
+import { FidgetConfig, FidgetSettings } from "@/common/fidgets";
+import React from "react";
+
+const PlusIcon = () => {
```

Comment:
Seriously, please make all of these Icons go in their own icons folder

Reply:
On it.